connectivity/common.h

```diff
@@ -16,6 +16,7 @@ extern "C" {
 
 #define STR_HTTP_NOT_FOUND "{\"message\": \"Request not found\"}"
 #define STR_HTTP_BAD_REQUEST "{\"message\": \"Invalid request header\"}"
+#define STR_HTTP_BAD_REQUEST_MESSAGE_OVERSIZE "{\"message\": \"In body 'message', requested message is too long.\"}"
```

Comment:
"Oversize" means something noticeably larger than usual, but it does not imply exceeding the limitation.